src/aap_eda/services/activation/manager.py

```diff
@@ -960,7 +964,16 @@ def monitor(self):
             self._unresponsive_policy()
             return
 
+        # last case is the success case
         if container_status.status == ActivationStatus.RUNNING:
+            if self.db_instance.status == ActivationStatus.WORKERS_OFFLINE:
+                LOGGER.info(
+                    f"Monitor operation: activation id: {self.db_instance.id} "
+                    "Activation is in WORKERS_OFFLINE state. "
+                    "Setting activation and instance status to running.",
+                )
+                self._set_activation_status(ActivationStatus.RUNNING)
```

Comment:
@Alex-Izquierdo Is this true even if the node is powered off? There won't be anything running at that point, is this setting it to running so that the Heartbeat check can mark it as Unresponsive.

Reply:
At this point the node can not be powered off, this code is executed in the node and we are setting the activation as running because its container is running.